tensorflow/training/docker/2.2.0/py3/cu101/Dockerfile.gpu

```diff
@@ -33,7 +33,7 @@ ARG ESTIMATOR_URL=https://aws-tensorflow-binaries.s3-us-west-2.amazonaws.com/est
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```

Comment:
Does the TF_URL on this dockerfile not need to be updated like the other dockerfiles?